src/main/java/com/dynatrace/prototype/payloadHandler/SlackHandler.java

```diff
@@ -83,14 +107,17 @@ public boolean receiveEvent(KeptnCloudEvent event) {
                 }
 
                 ChatPostMessageRequest request = SlackCreator.createPostRequest(event, channel, layoutBlocks, SLACK_NOTIFICATION_MSG);
-                ChatPostMessageResponse response = slack.methods(token).chatPostMessage(request);
+                OffsetDateTime requestKey = OffsetDateTime.parse(event.getTime());
 
-                if (response.isOk()) {
+                bufferedPostMessages.put(requestKey, request);
+
+                if(request.equals(bufferedPostMessages.get(requestKey))) {
+                    System.out.println("Post message added!");
```

Comment:
We should add a logging framework as a follow up task :)

Reply:
This is a future issue for probably the next internship.